sunbeamlib/initialize.py

```diff
@@ -31,9 +31,12 @@ def main():
         "--conda_fp", default=conda_fp, type=Path,
         help="path to conda (default: %(default)s)"
     )
+    parser.add_argument(
+        "--custom_configfile_fp", help="Custom config file absolute directory", 
```

Comment:
`custom_configfile_fp` is a little long. Could you change to something like `custom_config_fp` or even `template`? The `help` parameter should be something more like "Path to custom config file template, in YAML format"